lib/exq/adapters/queue/gen_server.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Exq.Adapters.Queue.GenServer do
```

Comment:
we could probably use a better name here instead of genserver, which doesn't communicate anything and kind of misleading as this is not a gen server.

Reply:
True. I could call it **Prod**, it would convey the environment as I called the other other one **Test**. Or maybe **Async** or  **Dispatch** ?

Reply:
looking at the sidekiq api https://github.com/mperham/sidekiq/wiki/Testing, we could probably call it based on the functionality. Like redis, memory (will just keep the jobs in memory and we could later do assert/clear the queue), inline (will call the worker directly)

Reply:
I could rename **Exq.Adapters.Queue.Test** to **Exq.Adapters.Queue.Fake** and **Exq.Adapters.Queue.GenServer** to **Exq.Adapters.Queue.Async** ?

Reply:
let's go with that